app/src/test/java/co/touchlab/droidconandroid/shared/presenter/UserDetailViewModelTest.java

```diff
@@ -35,7 +30,10 @@
     @Before
     public void setUp()
     {
-        factory = new UserDetailViewModel.Factory(interactor);
+        MockitoAnnotations.initMocks(this);
+        ObservableTransformer<UserAccount, UserAccount> transformer = upstream -> upstream.subscribeOn(
+                Schedulers.trampoline()).observeOn(Schedulers.trampoline());
```

Comment:
Whats the reasoning for using trampoline instead of immediate? It would seem trampoline would increase the risk of an issue because it does do some amount of scheduling.

Alternatively, what if the transformer just returned "upstream". ie, didnt add any schedulers at all. wouldnt that have the same effect?